src/utils_project.js

```diff
@@ -563,6 +563,55 @@ go.utils_project = {
 
 // SUBSCRIPTION HELPERS
 
+    get_subscription_msg_type: function(im, mother_id) {
+      // Look up what type of messages the mother is receiving
+
+        // get subscription
+        return go.utils
```

Comment:
I think if you do: https://github.com/praekelt/seed-stage-based-messaging/issues/7 first and use a FK serializer this would be a lot less API calls